src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -45,27 +45,31 @@ class CompanyStep extends React.Component {
             hasNoConnectionToCannabis: lodashGet(props, ['achData', 'hasNoConnectionToCannabis'], false),
             password: '',
         };
+
+        // These fields need to be filled out in order to submit the form
+        this.requiredFields = [
+            'companyName',
+            'addressStreet',
+            'addressCity',
+            'addressState',
+            'addressZipCode',
+            'website',
+            'companyTaxID',
+            'incorporationDate',
+            'industryCode',
+            'password',
+        ];
     }
 
     /**
      * @returns {Boolean}
      */
     validate() {
-        if (!this.state.password.trim()) {
```

Comment:
Should we leave these in?
B/c the button will be disabled, but what if someone hits return or  "Done" on mobile or whatever, won't that still submit the form and bypass the button being disabled?

Reply:
Basically I don't think we should rely on the button being enabled/disabled as a sign that the input is valid or not.